js/main.js

```diff
@@ -0,0 +1,174 @@
+      }
+      fragment.appendChild(item || photosTemplate);
+    }
+    return fragment;
+  }
+
+  function renderRentInfo(parameter) {
+    var offerCard = selectCardTemplate.cloneNode(true);
+    var photosWrapper = offerCard.querySelector('.popup__photos');
+    var example = '';
+    // Вывод названий
+    switch (parameter.offer.type) {
+      case 'palace':
+        example = 'Дворец';
+        break;
+      case 'flat':
+        example = 'Квартира';
+        break;
+      case 'house':
+        example = 'Дом';
+        break;
+      case 'bungalo':
+        example = 'Бунгало';
+        break;
+    }
+    // Выведите заголовок объявления offer.title в заголовок .popup__title.
+    offerCard.querySelector('.popup__title').textContent = parameter.offer.title;
+    // Выведите адрес offer.address в блок .popup__text--address.
+    offerCard.querySelector('.popup__text--address').textContent =
+      parameter.offer.address;
+    // Выведите цену offer.price в блок .popup__text--price строкой вида {{offer.price}}₽/ночь. Например, 5200₽/ночь.
+    offerCard.querySelector('.popup__text--price').textContent =
+      parameter.offer.price + '₽/ночь.';
+    // В блок .popup__type выведите тип жилья offer.type: Квартира для flat, Бунгало для bungalo, Дом для house, Дворец для palace.
+    offerCard.querySelector('.popup__type').textContent = example;
+    // Выведите количество гостей и комнат offer.rooms и offer.guests в блок .popup__text--capacity строкой вида {{offer.rooms}} комнаты для {{offer.guests}} гостей. Например, 2 комнаты для 3 гостей.
+    offerCard.querySelector('.popup__text--capacity').textContent =
+      parameter.offer.rooms + ' комнаты для ' + parameter.offer.guests + ' гостей';
+    // Время заезда и выезда offer.checkin и offer.checkout в блок .popup__text--time строкой вида Заезд после {{offer.checkin}}, выезд до {{offer.checkout}}. Например, заезд после 14:00, выезд до 12:00.
+    offerCard.querySelector('.popup__text--time').textContent =
+      'Заезд после ' +
+      parameter.offer.checkin +
+      ' , выезд до ' +
+      parameter.offer.checkout;
+    // В список .popup__features выведите все доступные удобства в объявлении.
+    offerCard.querySelector('.popup__features').textContent = parameter.offer.features;
+    // В блок .popup__description выведите описание объекта недвижимости offer.description.
+    offerCard.querySelector('.popup__description').textContent =
+      parameter.offer.description;
+    // В блок .popup__photos выведите все фотографии из списка offer.photos. Каждая из строк массива photos должна записываться как src соответствующего изображения.
+    offerCard.appendChild(addExtraPhotos(photosWrapper, parameter.offer.photos));
+    // Замените src у аватарки пользователя — изображения, которое записано в .popup__avatar — на значения поля author.avatar отрисовываемого объекта.
+    offerCard
+      .querySelector('.popup__avatar')
+      .setAttribute('src:', parameter.author.avatar);
```

Comment:
```suggestion
      .setAttribute('src', parameter.author.avatar);
```

а вообще можно и `offerCard.querySelector('.popup__avatar').src = parameter.author.avatar;`